cv19ResSupportV3.Tests/V3/E2ETests/CreateHelpRequestCall.cs

```diff
@@ -41,12 +43,26 @@ public async Task CreateHelpRequestCallReturnsTheCorrectInformation()
             Console.WriteLine("123456");
             Console.WriteLine(content);
             var stringContent = await content.ReadAsStringAsync().ConfigureAwait(true);
-            var convertedResponse = JsonConvert.DeserializeObject<HelpRequestCallCreateResponse>(stringContent);
+            // var convertedResponse = JsonConvert.DeserializeObject<HelpRequestCallCreateResponse>(stringContent);
```

Comment:
nit pick, I think we can remove these lines and change from async to void return type?

Reply:
I've removed those lines, thanks.